cymetric/brightlite_root_metrics.py

```diff
@@ -0,0 +1,10 @@
+from cymetric.root_metrics import root_metric
+
+agentstate_brightlite_fuelfabfacilityinfo = root_metric(
```

Comment:
Do we have a way to test these?  (This is part of the problem with them being distributed with Cymetric)